src/System.Net.Security/src/project.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "System.Diagnostics.Contracts": "4.0.0-*",
+    "System.Threading": "4.0.0-*",
+    "System.Net.Primitives": "4.0.10-beta-*"
```

Comment:
Replacing with 4.0.10-*" will break the build at this time although not because of Primitives types missing, but X509Certificate types. 

Since the dependency tree is different, I'd like to wait for @ericstj's answer on this before any further changes.

My current plan is to review all references once we enable individual packages for System.Net contracts.